### ISSUE TITLE
Localize function t doesn't work in Component

### ISSUE THREAD

**Issue by ungit** [xitrum-framework/xitrum#535]
Localize function `t` doesn't work in xitrum Component,  language always "en"
https://github.com/xitrum-framework/xitrum/blob/master/src/main/scala/xitrum/I18n.scala#L47 (Line 47)

Reproduce: 

```
package mypackage

import xitrum.{FutureAction, Component}
import xitrum.annotation.GET

class CompoWithoutView extends Component {
  def render() = {
    t("Hello World")
  }
}

@GET("foo/bar")
class MyAction extends FutureAction {
  def execute() {
    language = "ru"
    respondView()
  }
}
```

MyAction.jade:

```
- import mypackage._

!= newComponent[CompoWithoutView]().render()
```

**Comment by ngocdaothanh** [comment]
I've just fixed the problem via https://github.com/xitrum-framework/xitrum/commit/236f2e80b69dc3b90da668ec837daffcf023ddd3. I'll release a new Xitrum version in a few hours. Sorry for the inconvenience.

**Comment by ngocdaothanh** [comment]
@ungit Andrey Ustinov, thanks for reporting this problem, I've released Xitrum 3.24.0:
https://groups.google.com/forum/#!topic/xitrum-framework/RHv1cgXQAHI
